app/views/shared/_search_sort_select_section.html.erb

```diff
@@ -1,8 +1,12 @@
+<% label_text = '' %>
+<%
+  label_text = 'Sort by' if local_assigns[:is_crh_show]
+%>
```

Comment:
Can we do a ternary here? It would remove the need for setting the variable value twice. Maybe something like:
```
<% label_text =  local_assigns[:is_crh_show] ? 'Sort by' : '' %>
```

Reply:
Yep - got it.